src/main/java/frc/robot/Constants.java

```diff
@@ -56,7 +56,7 @@ public static final class SwerveModule {
 
     /** Calculations required for driving motor conversion factors and feed forward */
     public static final double DRIVING_MOTOR_FREE_SPEED_RPS = NeoMotor.FREE_SPEED_RPM / 60,
-        WHEEL_DIAMETER_METERS = Units.inchesToMeters(3) * 0.979,
+        WHEEL_DIAMETER_METERS = Units.inchesToMeters(3) * 0.979 * 0.9177,
```

Comment:
There should be a comment about what the 0.9177 is.

Reply:
Added